app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  validates :body, presence: true, length: { in: 2..20 }
+
+  belongs_to :posts
+  belongs_to :user_id
```

Comment:
**REQUIRED**
* Let's change `belongs_to : user_id` to `belongs_to :user`.
This will enable us to create a comment using a `user` instance like this
```ruby
comment = user.comments.create
```

Reply:
i would intend to create an index to refer to a particular user(the one that wrote that particular comment, also for future references if i want to visualize only comments for a specific user);
i have to run a migration to change the table right?